### ISSUE TITLE
Insufficient Funds Error -6 Windows Server 2016

### ISSUE THREAD

**Issue by eketubol** [coinfoundry/miningcore#490]
## Note
Please use the following format to file an issue. Issues that fail to comply will delay our responses.
Please remove this section when you submit the issue. Thank you for your cooperation.

### Description

I am currently using Windows Server 2016 as my operating system and i have noticed that payments are not being made. I have sent funds to the pool wallet to ensure there is enough funds to send the initial payment. What I am thinking is the issue might be because I am using the default QT wallet? Although it is able to read and pickup data from the wallet such as confirmation of blocks. Is there a step I am missing here? Do I need to enable something to allow miningcore to send funds from a QT wallet?

### Expected behavior

Payment sent without error

### Actual behavior

[2018-11-26 10:50:34.5344] [I] [PayoutManager] Processing payments for pool ravencoin
[2018-11-26 10:50:34.5344] [I] [PayoutManager] No updated blocks for pool ravencoin
[2018-11-26 10:50:34.5344] [I] [PayoutManager] No balances over configured minimum payout for pool ravencoin
[2018-11-26 10:50:34.5503] [I] [PayoutManager] Processing payments for pool stonecoin
[2018-11-26 10:50:34.5503] [I] [PayoutManager] No updated blocks for pool stonecoin
[2018-11-26 10:50:34.5503] [I] [stonecoin] [Bitcoin Payout Handler] Paying out 21.62539 STONE to 1 addresses
[2018-11-26 10:50:34.6832] [I] [ravencoin] Detected new block 465266 via RPC polling
[2018-11-26 10:50:34.6920] [I] [ravencoin] Broadcasting job
[2018-11-26 10:50:35.5764] [E] [stonecoin] [Bitcoin Payout Handler] sendmany returned error: Account has insufficient funds code -6

### Reproduces how often

100%

### Version

If you are using released version, please state your version.
If you are using clone from `dev` branch, please state commit hash of your copy.
Also, please include the OS name and version you're currently running.
I am currently running the most updated version of mining core from what I believe, I git cloned this repo a few days ago but then downloaded the latest release from github last night. 
I am using Windows Server 2016/

### Additional Information

Any additional information, log, configuration or data that might be necessary to reproduce the issue.

**Comment by oliverw** [comment]
Please run <code>yourcoin-cli getaddressesbyaccount ""</code> and make sure that the <code>address</code> value in your Miningcore config file matches one of those addresses.

**Comment by eketubol** [comment]
I don't have any cli.exe for stonecoin or raven. I only have the QT. Is that part of the problem? I can only seem to find QT wallets for Windows and no coind or coin-cli alternatives

**Comment by oliverw** [comment]
Run the command exactly like I wrote it: <code>getaddressesbyaccount ""</code>

**Comment by eketubol** [comment]
Okay thats given a different address, SQmzPdu2Njit1fd7Co8SchAYg9BJkignmQ

**Comment by oliverw** [comment]
Use that as pool address in the config and you should be fine.

**Comment by eketubol** [comment]
I've transfer the amount trying to be sent to the pool address but it is still saying insufficient funds. Do I need to mine the coin with the updated wallet information for it to take affect again?

**Comment by Xerxes7777** [comment]
Hi thanks for your help in advance.
I am having similar issue. I am using QT wallet for the coin Daemon too. 
I am running only one miner (solo mining) to get familiar with stratum.  It is mining ok and depositing the mined coin in the wallet address specified.

I put in getaddressesbyaccount "" in the QT Wallet command line I did not get any address 
I got the following:



02:48:44 |   | getaddressesbyaccount ""
[
]

Should it have an address. I have a vallet pool address in Config.json
Wallet password is unlocks the wallet.

[2018-11-27 02:51:14.3284] [I] [Californium pool] [Bitcoin Payout Handler] Paying out 158.42802 CF to 1 addresses
[2018-11-27 02:51:14.3284] [I] [Californium pool] [Bitcoin Payout Handler] Unlocking wallet
[2018-11-27 02:51:14.4740] [E] [Californium pool] [Bitcoin Payout Handler] sendmany returned error: Account has insufficient funds code -6
[2018-11-27 02:51:18.2555] [I] [Californium pool] Broadcasting job

Thanks again
